content/pages/groups/north.md

```diff
@@ -1,91 +1,96 @@
 ---
-title: Support Groups for North Alabama
+title: North Alabama Support Groups
 slug: groups-north
-description: A collection of transgender and non-binary support groups from northern Alabama
+description: A collection of support groups in North Alabama for those included in the transgender umbrella.
 ---
 
-# Transgender North Alabama - Adult Support Group (Huntsville)
+# Trans North Alabama - Adult Support Group (Huntsville, 18+)
 
-<img src="/extra_static/trans_north_al_logo.png" width="500" />
+[<img src="/extra_static/tna-logo.png" width="50%" />][tna-discord]
 
-This group is hosted in Huntsville, and meets several times per month. We
-welcome all adult transgender and non-binary identifying people regardless of
-their physical or social state of transition. You must also be 18 or older to
-attend.
+This group is hosted in Huntsville and meets several times per month. We
+welcome all adult transgender, nonbinary, and gender nonconfirming people regardless of
+their physical or social state of transition.
+
+- **You must be 18 or older to attend.**
```

Comment:
It looks kinda odd as its own bullet. We generally don't have an issue with people under 18 attempting to come to group, so I don't think it needs to be highlighted extra.

```suggestion
This group is hosted in Huntsville and meets several times per month. We
welcome all adult transgender, nonbinary, and gender nonconfirming people regardless of
their physical or social state of transition. You must be 18 or older to attend.
```